.github/workflows/trivy-nightly-scan.yaml

```diff
@@ -0,0 +1,35 @@
+name: Trivy Nightly Scan
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+
+permissions: read-all 
+jobs:
+  nightly-scan:
+    name: Trivy Scan nightly
+    strategy:
+      fail-fast: false
+      matrix:
+        # maintain the versions of etcd that need to be actively
+        # security scanned
+        versions: [v3.5.6]
```

Comment:
Minor comment: there is only one option, so it seems there is no need to use matrix. Let's keep it as it's for now, and consider to improve it in the next step

Reply:
Sure @ahrtr, thank you 👍